var/spack/repos/builtin/packages/py-ctranslate2/package.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyCtranslate2(CMakePackage, PythonExtension, CudaPackage):
+    """CTranslate2 is a C++ and Python library for efficient inference with
+    Transformer models."""
+
+    homepage = "https://github.com/OpenNMT/CTranslate2"
+    git = "https://github.com/OpenNMT/CTranslate2.git"
+
+    maintainers("meyersbs", "aweits")
+
+    version("3.12.0", tag="v3.12.0", submodules=True)
+
+    extends("python")
+
+    # From setup.py, CMakeLists.txt, requirements.txt
+    depends_on("py-setuptools", type="build")
+    depends_on("cmake@3.15:", type="build")
+    depends_on("py-pip", type="build")
+    depends_on("py-pybind11", type=("build", "run"))
```

Comment:
requirements.txt says 2.10.1. Any idea if this specific version is required?

Reply:
I can't find any info on this. I didn't run into any issues installing with an older version of py-pybind11. Thoughts?